### PR TITLE
[Core][Promotion][Taxonomy] Unify decorated services ids

### PR DESCRIPTION
**Pull request by Zales0123**
| Q               | A
| --------------- | ---
| Bug fix?        | yes |
| New feature?    | no |
| BC breaks?      | yes |
| Related tickets | |
| License         | MIT |

### CODE REVIEW

**Comment by pjedrzejewski** [comment]
Thank you Mateusz!

File: src/Sylius/Bundle/CoreBundle/Resources/config/services/promotion.xml [hunk 1]
```diff
@@ -13,11 +13,11 @@
...
     <services>
-        <service id="sylius.factory.promotion_action.decorated" class="Sylius\Component\Core\Factory\PromotionActionFactory" decorates="sylius.factory.promotion_action" public="false">
-            <argument type="service" id="sylius.factory.promotion_action.decorated.inner" />
+        <service id="sylius.custom_factory.promotion_action" class="Sylius\Component\Core\Factory\PromotionActionFactory" decorates="sylius.factory.promotion_action" public="false">
+            <argument type="service" id="sylius.custom_factory.promotion_action.inner" />
         </service>
-        <service id="sylius.factory.promotion_rule.decorated" class="Sylius\Component\Core\Factory\PromotionRuleFactory" decorates="sylius.factory.promotion_rule" public="false">
-            <argument type="service" id="sylius.factory.promotion_rule.decorated.inner" />
+        <service id="sylius.custom_factory.promotion_rule" class="Sylius\Component\Core\Factory\PromotionRuleFactory" decorates="sylius.factory.promotion_rule" public="false">
```
Comment (michalmarcinkowski) [inline src/Sylius/Bundle/CoreBundle/Resources/config/services/promotion.xml]: But why "**custom**_factory"? Wouldn't it be better to call it more specifically e.g. "**core**_factory"?

Comment (lchrusciel) [inline src/Sylius/Bundle/CoreBundle/Resources/config/services/promotion.xml]: I would vote for "**decorated**_factory"

Comment (michalmarcinkowski) [inline src/Sylius/Bundle/CoreBundle/Resources/config/services/promotion.xml]: For me `sylius.factory.promotion_rule.decorated` was fine.

Comment (Zales0123) [inline src/Sylius/Bundle/CoreBundle/Resources/config/services/promotion.xml]: According to [Symfony documentation](http://symfony.com/doc/current/service_container/service_decoration.html) we should keep convention like ``decorating_factory``. I've changed them to ``custom_factory`` as it was done this way in most cases.
However ``decorating_factory`` or ``custom_factory`` doesn't says too much about the service - maybe in fact we should think about something more sophisticated.

Comment (lchrusciel) [inline src/Sylius/Bundle/CoreBundle/Resources/config/services/promotion.xml]: Why? For me `sylius.decorating_factory.promotion_rule` seems to be sophisticated enough.

Comment (pjedrzejewski) [inline src/Sylius/Bundle/CoreBundle/Resources/config/services/promotion.xml]: I'd follow Symfony's convention. 👍  I did not know there is any.

**Pull request merged**